lib/manager/components/DeploymentViewer.js

```diff
@@ -348,8 +402,28 @@ export default class DeploymentViewer extends Component<Props, State> {
             </Panel>
           </Col>
           <Col sm={3}>
+            {/* Current deployment panel */}
+            <CurrentDeploymentPanel
+              deployJob={deployJob}
+              server={serverDeployedTo}
+              fetchDeployment={this.props.fetchDeployment}
+              deployment={deployment} />
+            {/* Configurations panel */}
             <Panel header={<h3><Icon type='cog' /> OTP Configuration</h3>}>
               <ListGroup fill>
+                <ListGroupItem>
+                  <Checkbox
+                    checked={deployment.r5}
+                    onChange={this._onChangeR5}>Use R5</Checkbox>
+                  <Checkbox
+                    checked={deployment.buildGraphOnly}
+                    onChange={this._onChangeBuildGraphOnly}>Build graph only</Checkbox>
+                  {deployment.r5 ? 'R5' : 'OTP'} version
+                  <Select
+                    value={deployment.r5 ? deployment.r5Version : deployment.otpVersion}
+                    onChange={this._onUpdateVersion}
+                    options={options ? options.map(v => ({value: v, label: v})) : []} />
```

Comment:
Would be nice to set the build bucket so we can use OTP builds from the IBI fork.

Reply:
As in, set the build bucket in the config file or by some other means? Do we have a way to write IBI builds to a bucket?

Reply:
Yeah, we should probably be writing IBI OTP builds to a separate bucket. Not sure how we'd add that bucket to datatools. Might be a good idea to have it be a config item I suppose.